bitbots_docs/docs/manual/tutorials/pycharm-ros.rst

```diff
@@ -15,7 +15,7 @@ If you use `Source ROS`, this is not necessary.
 5. Add --> `<catkin_ws>/devel/lib/python2.7/dist-packages`
```

Comment:
These two occurrences of `python2.7` can be replaced by something like `python3.x` if you are using only py3 packages, or not?

Reply:
i think so

Reply:
It should be `python3` and not `python3.x`